src/interfaces/Product.ts

```diff
@@ -0,0 +1,6 @@
+export interface ProductItemElements {
+  id: number;
+  image?: string;
```

Comment:
if it is a string then I would call the property "imageURI" (uri not url). Also have you checked if ProductItem will render correctly without an image?

Reply:
Image is required so i changed it